_posts/en/pages/2016-01-15-lifecycle.md

```diff
@@ -14,38 +14,38 @@ This document describes the life-cycle of the Bitcoin Core software package rele
 
 ## Versioning
 
-Bitcoin Core releases are versioned as follows: 0.MAJOR.MINOR, and release candidates are suffixed with rc1, rc2 etc.
+Bitcoin Core releases are versioned as follows: MAJOR.MINOR, and release candidates are suffixed with rc1, rc2 etc.
 
 ## Major releases
 
 We aim to make a major release every 6-7 months.
 
-These will be numbered 0.11.0, 0.12.0 etc.
+These will be numbered 22.0, 23.0 etc.
 
 ## Maintenance releases
 
 We will provide maintenance "minor releases" that fix bugs within the major releases. As a general rule we do not introduce major new features in a maintenance release (except for consensus rules). However, we may add minor features where necessary, and we will back-port consensus rule changes such as soft forks.
 
-Minor releases will be numbered 0.11.1, 0.11.2, 0.12.1, 0.12.2 etc.
+Minor releases will be numbered 22.1, 22.2, 23.1, 23.2 etc.
 
 ## Consensus rules
 
-Proposals to change consensus rules are always shipped first in maintenance versions such as 0.11.2, 0.12.1 etc. This makes it easier for enterprise users to assess and test the proposal because of its smaller changeset compared to a major release. It also allows users who follow a more conservative upgrade path to adopt consensus rule changes in a more timely manner.
+Proposals to change consensus rules are always shipped first in maintenance versions such as 22.2, 23.1 etc. This makes it easier for enterprise users to assess and test the proposal because of its smaller changeset compared to a major release. It also allows users who follow a more conservative upgrade path to adopt consensus rule changes in a more timely manner.
 
 ## Maintenance period
 
 We maintain the major versions until their "Maintenance End". We generally maintain the current and previous major release.
-So if the current release is 0.13, then 0.12 is also considered maintained.
-Once 0.14 is released, then 0.12 would be considered at its "Maintenance End".
+So if the current release is 23.0, then 22.0 is also considered maintained.
+Once 24.0 is released, then 22.0 would be considered at its "Maintenance End".
 The older the major release, the more critical issues have to be to get backported to it, and the more to warrant a new minor release.
```

Comment:
This phrase is a bit unclear to me: is this a better suggestion, or am I misunderstanding?
```suggestion
The older the major release, the more critical issues have to be to get backported to it, and the more a new minor release is warranted.
```

Reply:
Okay. I still think it's a pretty confusing phrase. I would intuitively interpret this as new minor releases being _more_ warranted/frequent for older major releases, which is pretty much the opposite. Out of scope, so feel free to ignore, but I think this says the same thing in more simple and clear terms.

```suggestion
As a major release becomes older, issues have to be increasingly critical to get backported to it, and an increasing amount or severity of issues is required to warrant a new minor release.
```

Reply:
Ok. Have taken the newer wording.